packages/core/helper-plugin/lib/src/components/ContentBox/index.js

```diff
@@ -15,25 +16,43 @@ const TypographyWordBreak = styled(Typography)`
   word-break: break-all;
 `;
 
-const ContentBox = ({ title, subtitle, icon, iconBackground, endAction, titleEllipsis }) => {
+const ContentBox = ({
+  title,
+  subtitle,
+  icon,
+  iconBackground,
+  endAction,
+  titleEllipsis,
+  customIcon,
+  decorator,
+}) => {
   if (title.length > 70 && titleEllipsis) {
     title = `${title.substring(0, 70)}...`;
   }
 
   return (
-    <Flex shadow="tableShadow" hasRadius padding={6} background="neutral0">
-      <IconWrapper background={iconBackground} hasRadius padding={3}>
-        {icon}
-      </IconWrapper>
+    <Flex shadow="tableShadow" hasRadius padding={6} background="neutral0" position="relative">
+      {customIcon || (
+        <IconWrapper background={iconBackground} hasRadius padding={3}>
+          {icon}
+        </IconWrapper>
+      )}
       <Stack spacing={endAction ? 0 : 1}>
         <Flex>
           <TypographyWordBreak fontWeight="semiBold" variant="pi">
             {title}
           </TypographyWordBreak>
           {endAction}
         </Flex>
-        <Typography textColor="neutral600">{subtitle}</Typography>
+        {decorator ? (
+          <Box width="80%">
```

Comment:
I think we should remove this now that there's no vertical overlap between the flag and the text?

I think it's also odd because "decorator" is quite generic and this adjustment is very specific to the flags. And it's a percentage whereas the size of the flags is fixed, so this adjustment was tied to the viewport width